java/runtime/src/main/java/io/ray/runtime/config/RayConfig.java

```diff
@@ -166,8 +174,13 @@ public RayConfig(Config config) {
     if (StringUtils.isNotBlank(bootstrapAddress)) {
       setBootstrapAddress(bootstrapAddress);
     } else {
-      // We need to start gcs using `RunManager` for local cluster
+      // We will try to get address from env, If it still empty,
+      // we need to start gcs using `RunManager` for local cluster
       this.bootstrapAddress = null;
+      String bootstrapAddressEnv = System.getenv("RAY_ADDRESS");
```

Comment:
Why do we need `RAY_ADDRESS`? I think the address should be passed by configuration instead of env var, like:
```java
java -cp xxxx -Dray.address="100.19.21.1:7743" mainClass
```